src/main/resources/templates/client/contact.html

```diff
@@ -9,13 +9,20 @@
     <div class="row">
         <div class="col-sm-12">
             <h1>Kontakt</h1>
-            <h2>Kontaktní osoba</h2>
-            <p th:text="${contact.name}">Jmeno</p>
-            <h2>Telefon</h2>
-            <p th:text="${contact.phoneNumber}" align="center">Telefon</p>
+            <h2 th:text="${contact.name}">Jmeno</h2>
+            <ul class="contact">
+                <li>
+                    <i class="fas fa-phone"></i>
+                    <span th:text="${contact.phoneNumber}">email@email.com</span>
```

Comment:
vim ze je to jen placeholder ale je to matoucí u telefonniho cisla :)
.....>email@email.com<...